test/metabase/api/alert_test.clj

```diff
@@ -599,11 +599,14 @@
                               :aggregation  [["count"]]
                               :breakout     [[:field (mt/id :checkins :date) {:temporal-unit :hour}]]}}})
 
-(defn- alert-question-url [card-or-id]
-  (format "alert/question/%d" (u/the-id card-or-id)))
+(defn- alert-question-url [card-or-id & [archived]]
+  (if archived
+    (format "alert/question/%d?archived=%b" (u/the-id card-or-id) archived)
+    (format "alert/question/%d" (u/the-id card-or-id))))
 
-(defn- api:alert-question-count [user-kw card-or-id]
-  (count ((alert-client user-kw) :get 200 (alert-question-url card-or-id))))
+(defn- api:alert-question-count [user-kw card-or-id & [archived]]
+  (def my-archived archived)
```

Comment:
Do we need this? clj-kondo is probably going to be fussy if you do this

Reply:
Oops, that was for debugging, sorry